astropy_helpers/commands/build_sphinx.py

```diff
@@ -120,7 +120,10 @@ def run(self):
         build_cmd_path = os.path.abspath(build_cmd.build_lib)
 
         ah_importer = pkgutil.get_importer('astropy_helpers')
-        ah_path = os.path.abspath(ah_importer.path)
+        if ah_importer is None:
```

Comment:
what prompted this change?

Reply:
I think this was actually related to trying to build the docs with ``python setup.py build_docs --use-system-astropy-helpers`` so isn't quite related to this PR but I found it while trying to use this PR